pom.xml

```diff
@@ -154,6 +154,11 @@
                 <artifactId>org.apache.felix.framework</artifactId>
                 <version>5.6.2</version>
             </dependency>
+            <dependency>
```

Comment:
Remove `commons-lang` as it is only used for one method that easily can be rewritten to be better. Also `commons-lang` is code from 2011. Bleh. :mask: